.github/workflows/tests.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           globalPackages: http-server
 
       - name: Install Playwright
-        run: npx playwright@1.20.0 install --with-deps
+        run: npx playwright@1.27.1 install --with-deps
```

Comment:
I think I've added something about this in the README or CONTRIBUTING file. Could you check if these need an update too?

Reply:
Updated 👍🏼